libs/core/execution/include/hpx/execution/queries/read.hpp

```diff
@@ -87,7 +92,7 @@ namespace hpx::execution::experimental {
             template <typename Tag1>
             friend auto tag_invoke(
                 get_completion_signatures_t, read_sender<Tag1>, no_env)
-                -> dependent_completion_signatures<no_env>;
+                -> std::execution::dependent_completion_signatures<no_env>;
```

Comment:
this doesn't look right, is std::execution always available?

Reply:
You are right, it is not and this is wrong. Also, this part of the code is ifdef-ed away so I am reverting it to the pre-stdexec code.